firebase-firestore/ktx/src/test/java/com/google/firebase/firestore/TestUtil.java

```diff
@@ -138,36 +138,6 @@ public static QuerySnapshot querySnapshot(
     return new QuerySnapshot(query(path), viewSnapshot, FIRESTORE);
   }
 
-  /**
```

Comment:
Thanks for doing this. At some point we should probably cut these down to the minimum, given the assumption there's no reasonable way to share with the main Firestore testutil.

Reply:
FWIW, only half of the methods are currently used, but it doesn't seem trivial to just put them in the test class (so I refrained from doing this for now).